ext/compressors/zstd/zstd_compress.c

```diff
@@ -89,6 +112,68 @@ zstd_error(WT_COMPRESSOR *compressor, WT_SESSION *session, const char *call, siz
     return (WT_ERROR);
 }
 
+/*
+ * zstd_alloc_context --
+ *     WiredTiger Zstd allocate context from context pool.
+ */
+static void
+zstd_alloc_context(
+  ZSTD_COMPRESSOR *zcompressor, WT_SESSION *session, CONTEXT_TYPE ctx_type, ZSTD_CONTEXT **context)
```

Comment:
For the `context` parameter - we'd usually name that `contextp` - since it's a pointer to a context structure.